package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Conditional expresstion util for server/client",
   "main": "./dist/index.js",
   "scripts": {
-    "test": "gulp &&./node_modules/.bin/mocha --reporter spec",
-    "prepublish": "gulp"
+    "test": "mocha --compilers js:babel-core/register --reporter spec",
```

Comment:
This will automatically look inside `./node_modules/.bin` after `mocha` and execute it. Also allows for cross platform usage.